bin/rebuild-store.py

```
@@ -408,6 +408,39 @@ def promote(build: dict[str, Any], store_dir: Path) -> dict[str, Any]:
+        return "off (no semantic-on)"
+    venv_py = store_dir / "venv" / "bin" / "python"
+    interpreter = str(venv_py) if venv_py.exists() else sys.executable
+    cmd = [
+        interpreter, str(BIN_DIR / "embed-facts.py"),
+        "--facts", str(store_dir / "facts.json"),
+        "--sidecar", str(store_dir / "embeddings.npz"),
+        "--model-dir", str(store_dir / "model"),
+    ]
+    try:
+        # nosec B603 - fixed interpreter + sibling bin/ script, no untrusted input
+        proc = subprocess.run(cmd, capture_output=True, text=True, timeout=1800, check=False)  # nosec B603
```

Ruff (0.16.1)

[error] `subprocess` call: check for execution of untrusted input

(S603)